tests/try_scraper/test_try_scraper.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import unittest
+import urllib
+
+from mozdownload import TryScraper
+from mozdownload.utils import urljoin
+import mozhttpd_base_test as mhttpd
+
+firefox_tests = [
```

Comment:
Could you please put the parameters for `args` onto a separate line each?

For readability reasons and to be inline with the other tests.